README.md

```diff
@@ -60,6 +60,8 @@ bundle exec rails parallel:spec
 bundle exec rails parallel:spec[number]
 ```
 
+Specs are run in parallel on CI (2 cores).
```

Comment:
@rizzkhan1 can we get more cores?

Reply:
this statement is likely to become stale, I'm not sure I'd add this line personally

Reply:
@defong - so apparently we have no control of the underlying hardware for an Azure agent. Another way we could potentially see further time saving is to utilise more agents however having discussed with @misaka , this will make setting up parallel testing a lot more complicated (more dbs (or namespaces?) will be required) and may not be worth the additional effort. Will be interesting to see what the improvement is on backend CI utilising 2 cores

Reply:
I've removed the update to README @timabell